Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -1380,6 +1427,12 @@ private extension CustomerCenterViewModelTests {
         )
     }()
 
+    static let customerInfoWithoutSubscriptions: CustomerInfo = {
```

Comment:
this is not used. Did you intend to use it in https://github.com/RevenueCat/purchases-ios/pull/5614/files#diff-a2c613be0d52736ad1a27bda275d71df7e8d16c26bebb96611ff09724d3ad8adR1234?